tests/e2e/tests/cart/cart-block.merchant.block_theme.spec.ts

```diff
@@ -185,6 +186,24 @@ test.describe( 'Merchant → Cart', () => {
 						' [data-type="woocommerce/empty-cart-block"]'
 				)
 			).toBeHidden();
+
+			await editor.selectBlocks(
+				await editorUtils.getBlockByName(
+					'woocommerce/cart-order-summary-shipping-block'
+				)
+			);
+			await editor.openDocumentSettingsSidebar();
+			const shippingLabel = editorUtils.page.getByLabel(
+				'Shipping calculator'
+			);
+			await shippingLabel.check();
+			await expect(
+				editor.canvas.getByText( 'Change address' )
+			).toBeVisible();
+			await shippingLabel.uncheck();
+			await expect(
+				editor.canvas.getByText( 'Change address' )
+			).toBeHidden();
```

Comment:
Could you clarify why there isn't a separate test for the `toggle shipping calculator`? The existing test is described as `shows empty cart when changing the view`, which doesn't seem to correspond with the action of toggling a shipping calculator.

Reply:
Yes, we are trying to combine tests where possible, I feel like it makes sense to combine these tests as they're both using attributes to hide/show certain elements of the block.

I updated the name to be clearer.